modules/Bio/EnsEMBL/Compara/PipeConfig/ProteinTrees_conf.pm

```diff
@@ -85,6 +85,7 @@ sub default_options {
         'fasta_dir'             => $self->o('work_dir') . '/blast_db',  # affects 'dump_subset_create_blastdb' and 'blastp'
         'cluster_dir'           => $self->o('work_dir') . '/cluster',
         'dump_dir'              => $self->o('work_dir') . '/dumps',
+        'gene_dumps_dir'        => $self->o('dump_dir') . '/genes',
```

Comment:
For all other types of file dump, I used the `dump_dir` as a base. It won't impact functionality, but I guess we should try to be consistent. 
We can either change `homology_dumps_dir` and `orthotree_dir` to use `work_dir` as a base or use `dump_dir` as a base here.

Reply:
Sorry, I don't understand. Yes `homology_dumps_dir` and `orthotree_dir` use `dump_dir`:
```perl
'homology_dumps_dir'        => $self->o('dump_dir') . '/homology_dumps/',
'orthotree_dir'             => $self->o('dump_dir') . '/orthotree/',
```
and so does `gene_dumps_dir` :thinking: ?

Reply:
Sorry, must have misread this. Really thought it was pointing at `$self->o('work_dir') . '/genes'` 🤦‍♀️